TidepoolOnboarding/Support/AttributedString.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  AttributedString.swift
+//  TidepoolOnboarding
+//
+//  Created by Darin Krauss on 5/10/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import Foundation
+
+struct AttributedString {
+    enum Attribute {
+        case bold
+        case italic
```

Comment:
While the current solution is only implemented to handle just what is needed now (bold, italic), it is designed such that it can be expanded to encompass any sort of attributes applied to the text. I could even imagine something like: `<font face="whatever" size="24" weight="semi-bold" color="red">`. If we decided that one `font` tag completely overrides another, then we'd add only one new element to the set. If, however, we decided that each `font` attribute could be applied independently, then we'd add one new element for each attribute. Either way, this solution should be expandable as needed.